src/components/layout/HeaderMenu.vue

```diff
@@ -12,21 +12,16 @@ export default {
   data() {
     return {
       menuItems: [
-        { label: this.$t("menu.contacts"), route: "/profiles/contacts" },
         {
-          label: this.$t("menu.trailsTreasuries"),
-          route: "/trails/treasuries",
+          label: this.$t("menu.daos"),
+          route: "/trails/ballots",
```

Comment:
```suggestion
          route: "/trails/treasuries",
```

Reply:
My bad! I miss understood the request.
The fix required more than just changing the URL so a did it in this commit [d867ddc](https://github.com/telosnetwork/app-telos-native/pull/88/commits/d867ddcfefd9a5e13c6f1d22f7e87b3b1156065f).